charts/stable/whoogle/values.yaml

```
@@ -0,0 +1,35 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: benbusby/whoogle-search
+  pullPolicy: IfNotPresent
+  # tag defaults to appVersion in Chart.yaml
+  # tag:
```

```suggestion
  tag: 0.4.0
```
Even though the common library will default to the appVersion, we still keep an explicit tag in our values.yaml, also so that it gets populated in the README.